src/Components/FadeInBox.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, BoxProps, useDidMount } from "@artsy/palette"
+
+interface FadeInBoxProps extends BoxProps {}
+
+export const FadeInBox: React.FC<FadeInBoxProps> = ({
+  children,
+  ...boxProps
```

Comment:
```suggestion
  ...rest
```

would be more idiomatic